app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -727,14 +727,16 @@ export default {
             case 'custom':
               if (obj.component.validate.custom.length) {
                 validatorIdentity +=
-                  '|custom:' + obj.component.validate[validity].trim();
+                  '|custom:' +
+                  obj.component.validate[validity].trim().replaceAll(',', '‚');
```

Comment:
This validation case is only meant for Bulk upload and this COMMA replacement is just for the CSV validation report. 

If a form owner doesn't export in CSV but only in JSON, will their commas be replaced with these fake commas?
Answer: No, There is only a CSV report available to download so commas in that report/csv will be replaced.

You won't be able to do a search for commas with this replacement, you'd have to search with the fake comma.
Answer: As CSV does not support commas as a context I have to replace it with a Unicode comma character that is Unicode Character “‚” (U+201A) / Single Low-9 Quotation Mark. So that a sentence will not break into the new line.

![image](https://github.com/bcgov/common-hosted-form-service/assets/107434397/b43cb4dc-cdc0-4128-a65a-7c260f40cf81)